src/content/docs/setup/analytics/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,165 @@
+
+### Required Identifiers
+
+* **IMS Organization ID**: Your Adobe organization identifier (format: `1234567890ABCDEF7F000101@AdobeOrg`)
+* **Datastream ID**: A configured datastream for routing data (format: `12345678-1234-1234-1234-123456789012`)
+
+### How to Find Your Identifiers
+
+**IMS Organization ID:**
+To locate your IMS Organization ID, refer to the <Link href="https://experienceleague.adobe.com/docs/core-services/interface/administration/organizations.html" text="Organizations documentation" />. You can typically find this in:
+
+- Adobe Admin Console
+- Developer Console
+- Any Adobe Experience Cloud application under Account Settings
+
+**Datastream ID:**
+Your datastream must be configured to route data to Adobe Experience Platform. For detailed instructions on creating and configuring a datastream, see the <Link href="https://experienceleague.adobe.com/docs/experience-platform/datastreams/overview.html" text="Datastreams documentation" />.
+
+## Configuration
+
+To enable data flow from your storefront to the Experience Platform Edge Network, you need to configure the Adobe Data Layer with your AEP credentials.
+
+<Tasks>
+
+<Task>
+### Locate your delayed script file
+
+Typically `scripts/delayed.js` in your project
+</Task>
+
+<Task>
+### Add the AEP configuration to your Adobe Data Layer initialization
+
+<CodeExample code={`
+window.adobeDataLayer.push(
+    {
+      aepContext: {
+        imsOrgId: '1234567890ABCDEF7F000101@AdobeOrg',
+        datastreamId: '12345678-1234-1234-1234-123456789012'
+      },
+    }, 
+    { 
+      eventForwardingContext: {
+        commerce: true,
+        aep: true
+      }
+    },
+);
+`} lang="js" title="scripts/delayed.js" />
+
+</Task>
+</Tasks>
+
+### Configuration Parameters
+
+| Parameter | Description | Required |
+|-----------|-------------|----------|
+| `aepContext.imsOrgId` | Your Adobe IMS Organization ID | Yes |
+| `aepContext.datastreamId` | Your configured datastream ID for routing data to AEP | Yes |
+| `eventForwardingContext.commerce` | Enables commerce event tracking | Yes |
+| `eventForwardingContext.aep` | Enables forwarding events to Adobe Experience Platform | Yes |
+
+### What This Configuration Does
+
+- **`aepContext`**: Provides the necessary identifiers for routing data to your specific Adobe Experience Platform environment
+- **`eventForwardingContext.aep: true`**: Instructs the Adobe Data Layer to forward commerce events to the Experience Platform Edge Network
+- **`eventForwardingContext.commerce: true`**: Enables tracking of commerce-specific events like product views, cart additions, and purchases
+
+## Data Flow and Events
+
+Once configured, your storefront will automatically send the following types of events to Adobe Experience Platform:
+
+- **Product interactions**: Views, clicks, add-to-cart actions
+- **Shopping behavior**: Cart updates, checkout steps, purchases
+- **User engagement**: Page views, time on site, navigation patterns
```

Comment:
time on site? - we don't have it
Here is a list of events we send to AEP https://experienceleague.adobe.com/en/docs/commerce/data-connection/event-forwarding/events
so we can add for customer profile such events as createAccount, editAccount, login, logout